framework/test/unit/engine/generator/strategies.spec.ts

```diff
@@ -185,6 +185,25 @@ describe('strategies', () => {
 				);
 			});
 
+			it('should not execute transaction if maximum size after maximum size is reached', async () => {
```

Comment:
I am not sure if the current description makes sense about what is being tested. Would be nice to make it a bit more clear.
```suggestion
			it('should not execute transaction if the transaction byte size exceeds max transaction byte size allowed for the block', async () => {
```